scripts/generate.sh

```diff
@@ -24,7 +24,7 @@ generate_sdk() {
 
 npx @openapitools/openapi-generator-cli version-manager set 5.2.0
 echo "Generating SDKs"
-additional_properties="generateInterfaces=true,enumClassPrefix=true"
+additional_properties="generateInterfaces=true,enumClassPrefix=true,structPrefix=true"
```

Comment:
Is there a reason why we are adding this. This change will have breaking changes to all types generated.

Reply:
I know, sorry about this, but otherwise the generator produces duplicate structs (and therefore compilation failures) for schemas that are using multiple tags per operation.

Reply:
This will make it safe, in the future, in case other schemas have multiple tags as well

Reply:
Well, I can now actually address it on the Apicurio Registry repo side if you prefer?

Reply:
Can you point out the schema that is causing the issue?

Reply:
Checkout this PR, then `git revert 241e90e1e617cc60e16b3292e46cc09d142ea25a` to remove the change, then `make generate`, you can see that, for example, `ApiListArtifactTypesRequest` struct is duplicated. It seems this code https://github.com/redhat-developer/app-services-sdk-go/blob/main/scripts/generate.sh#L79 is not working properly.

Reply:
I can remove the tags on the Registry side if that's ok, we've made some changes so we can adjust the schema we send

Reply:
Hey sorry, I think the best thing to do then is fix the process or keeping one tage in the schema that we were trying to do before.

Reply:
Ok, let me close this PR and reopen a new one